code.sh

```diff
@@ -60,6 +60,13 @@ for i in "${SDK[@]}"; do
   fi
 done
 
+# Create /etc/static/shells before than /etc/shells, because /etc/shells ls linkd etc/static/shells
+# when use nixos is default has't /etc/static/shells
+if [ ! -f /etc/static/shells ]; then
```

Comment:
Should perform a similar check as below to only create the link if it exists on the NixOS host

```sh
if [ ! -e /etc/static/shells ] && [ -e /var/run/host/etc/static/shells ]; then
```